candle-flash-attn/build.rs

```diff
@@ -57,9 +57,22 @@ fn main() -> Result<()> {
             #[allow(clippy::redundant_clone)]
             out_dir.clone()
         }
-        Ok(build_dir) => PathBuf::from(build_dir),
+        Ok(build_dir) =>
+        {
+            let path = PathBuf::from(build_dir);
+            let path = if path.is_relative() {
```

Comment:
Shouldn't canonicalize take care of making the path absolute? The doc mentions that it's similar to `realpath`.

Reply:
That's true. However, the error message, while true, is a bit misleading. When I first did that way, I was confused, for an embarassingly long time, why it complains, that "./target/libflash" does not exist, even though I've just created that directory. In hindsight, it was obvious, that the working directory of this script is "./candle-flash-attn" - and not the root of the repository - so if I want to store something under "./target", I need to specify as "../target/libflash". 
 I thought, about interpreting the relative path from the repository root, but I'm not sure, how much confusion that would occur.

Reply:
Maybe we should just tweak the error message to mention the current directory?

Reply:
Yes, that could work as well.